webapp/components/navbar_dropdown.jsx

```diff
@@ -291,7 +291,10 @@ export default class NavbarDropdown extends React.Component {
                         <li>
                             <a
                                 href='#'
-                                onClick={() => this.setState({showUserSettingsModal: true})}
+                                onClick={(e) => {
```

Comment:
Not important for this PR, but we should probably move to putting these in named functions from now on. That'll get ESLint to stop printing warnings about binding stuff in the render method